src/main/java/frc/robot/RobotContainer.java

```diff
@@ -304,7 +310,8 @@ private Command shootSpeaker() {
 
   private Command prepareToScoreSpeaker() {
     return Commands.parallel(
-        turretCommandFactory.getAimTurretCommand(), shooterCommandFactory.revShooter());
+        turretCommandFactory.moveUpwards(),
+        shooterCommandFactory.revShooter()); // TODO: just testing pitch from tunable number
```

Comment:
This was changed just for testing. What do we want here for San Antonio?

Reply:
I think the idea is that we get lime light working and then have the appropriate command there, but idk